### PR TITLE
Add required update parameter

### PR DESCRIPTION
**Pull request by xuzhang3**
## All Submissions:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.
* [x] All new and existing tests passed.
* [x] My code follows the code style of this project.
* [x] I ran lint checks locally prior to submission.
* [x] Have you checked to ensure there aren't other open PRs for the same update/change?

## What about the current behavior has changed?

Issue Number: #312  
Required parameters are not  send to service when update the service connection. Add new configuration to record the parameters.

## Does this introduce a change to `go.mod`, `go.sum` or `vendor/`?

- [ ] Yes
- [x] No

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Any relevant logs, error output, etc?

(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

### CODE REVIEW

**Comment by xuzhang3** [comment]
Test result:
image
image

File: azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_azurecr.go [hunk 1]
```diff
@@ -47,14 +69,19 @@ func expandServiceEndpointAzureCR(d *schema.ResourceData) (*serviceendpoint.Serv
 			"tenantId":           d.Get("azurecr_spn_tenantid").(string),
 			"loginServer":        loginServer,
 			"scope":              scope,
+			"serviceprincipalid": d.Get("spn_id").(string),
```
Comment (yupwei68) [inline azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_azurecr.go:72]: Since it's `computed`, why do we get its value?

Comment (xuzhang3) [reply azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_azurecr.go:72]: This parameters are required when we try to update the service connection

**Review by yupwei68 — APPROVED** [review]
LGTM！

**Pull request merged**